pkg/kubelet/status/status_manager.go

```diff
@@ -666,6 +666,10 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		klog.V(1).InfoS("Status for pod is up-to-date; skipping", "podUID", uid)
 		return
 	}
+	// If kubeClient == nil, we are running in standalone mode
```

Comment:
is this needed, I see that `Start()` returns early https://github.com/kubernetes/kubernetes/blob/9cf7b2e7a529aeaa9de9e20377b7a8a3dfa94d7f/pkg/kubelet/status/status_manager.go#L156-L163

is this called from another place?

Reply:
I added this mistakenly. This should not work as you pointed out.